backend/vcdat/Files.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+def isValidFile(start_path, item):
+    if os.path.isfile(os.path.join(start_path, item)) and item[0] != '.' and item.endswith('.nc'):
+        return True
+    return False
+
+def isValidDir(start_path, item):
+    if not os.path.isfile(os.path.join(start_path, item)) and item[0] != '.':
+        return True
+    return False
+
+def getFilesObject(start_path):
+    if start_path != '/':
+        name = start_path.split('/')[-2]
+    else:
+        name = '/'
+
+    cur_dir_items = {
+        'name': name,
+        'sub_items': {},
+        'directory': True,
+        'path': start_path
+    }
+    for item in os.listdir(start_path):
+        if isValidFile(start_path, item) or isValidDir(start_path, item):
```

Comment:
This can be re write with out the or, and our with out calling the check functions more than once.